apps/analytics/src/scripts/2_initial_aggregated_course_analytics.py

```
@@ -0,0 +1,43 @@
+# This script computes the aggregated course analytics
+# ! This script is a copy of the corresponding notebook content and needs to be kept in sync with it
+
+import os
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)